### PR TITLE
Fix a version from PGroonga 2.0.3 to PGroonga 2.0.2

### PR DESCRIPTION
**Pull request by komainu8**

### CODE REVIEW

File: en/_posts/2017-10-10-pgroonga-2.0.2.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-description: PGroonga (fast full text search module for PostgreSQL) 2.0.3 has been released!
+title: PGroonga (fast full text search module for PostgreSQL) 2.0.2 has been released
+description: PGroonga (fast full text search module for PostgreSQL) 2.0.2 has been released!
 ---
 
-## PGroonga (fast full text search module for PostgreSQL) 2.0.3 has been released
+## PGroonga (fast full text search module for PostgreSQL) 2.0.2 has been released
 
-[PGroonga](https://pgroonga.github.io/) 2.0.3 has been released! PGroonga makes PostgreSQL fast full text search for all languages.This is major version up!
+[PGroonga](https://pgroonga.github.io/) 2.0.2 has been released! PGroonga makes PostgreSQL fast full text search for all languages.This is major version up!
```
Comment (yoshihara) [inline en/_posts/2017-10-10-pgroonga-2.0.2.md]: We need missing space after `.` ?

Comment (yoshihara) [reply en/_posts/2017-10-10-pgroonga-2.0.2.md]: (Question) The previous PGroonga' version is 2.0.1, so isn't 2.0.2 major version ? 🤔 (Also in Japanese)

File: en/_posts/2017-10-10-pgroonga-2.0.2.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 
 ## About PGroonga
 
-I will explain about PGroonga. Because it is the first release announcement after the major version upgrade.The highkight is summarized after this.
+I will explain about PGroonga. Because it is the first release announcement after the major version upgrade.The highlight is summarized after this.
```
Comment (yoshihara) [inline en/_posts/2017-10-10-pgroonga-2.0.2.md]: ditto

**Review by yoshihara — COMMENTED** [review]
Would you please check my comments 🙏 thank you.

File: en/_posts/2017-10-10-pgroonga-2.0.2.md (hunk 1 above)
Comment (obache) [reply en/_posts/2017-10-10-pgroonga-2.0.2.md]: No public release announcement for 2.0.0 and 2.0.1 here, then 2.0.2 is the first announcement for 2.X major versin up.

Comment (yoshihara) [reply en/_posts/2017-10-10-pgroonga-2.0.2.md]: I see. I wondered If this sentence confused users with 2.0.0 or 2.0.1, but you don't worry about it, I think it's OK. 😃

**Comment by komainu8** [comment]
@yoshihara Thank you for your comment. I've fixed it in `aa99db45c427b4faf8b862ad5d705aab3405f183`

**Review by yoshihara — APPROVED** [review]
Thanks!

**Pull request merged**